connector-catalog/src/main/java/io/syndesis/connector/catalog/ConnectorCatalog.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2016 Red Hat, Inc.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
this might break the license check (wonder why it did not?)

Reply:
Seem like that markup doesn't harm for this plugin. I regenerated the header nevertheless.